ci-operator/step-registry/hypershift/aws/create-iam/hypershift-aws-create-iam-commands.sh

```
@@ -0,0 +1,62 @@
+
+set -exuo pipefail
+
+AWS_GUEST_INFRA_CREDENTIALS_FILE="/etc/hypershift-ci-jobs-awscreds/credentials"
+
+HC_REGION=${HYPERSHIFT_AWS_REGION:-$LEASED_RESOURCE}
+
+if [[ $HYPERSHIFT_GUEST_INFRA_OCP_ACCOUNT == "true" ]]; then
+  AWS_GUEST_INFRA_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+fi
+
+INFRA_ID="$(cat ${SHARED_DIR}/infra-id)"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)